samples/net/aws_iot/certs/ca-cert.pem

```diff
@@ -1,3 +1,3 @@
-"-----BEGIN CA CERTIFICATE-----\n"
-"-----CERTIFICATE-----\n"
-"-----END CA CERTIFICATE-----\n"
+"-----BEGIN CERTIFICATE-----\n"
+"-----certificate-----\n"
```

Comment:
Maybe we could make this even more explicit by saying some like `Replace this with the AWS IoT root CA` and similar for the other entries

Reply:
Please suggest changes

Reply:
I was thinking of just replacing `-----certificate-----\n` with `Replace this with the AWS IoT root CA`, but don't really have strong opinions. I'll approve as is.